sdk/common/src/test/java/io/opentelemetry/sdk/internal/ComponentRegistryTest.java

```diff
@@ -52,5 +61,36 @@ void get_DifferentInstance() {
         .isNotSameAs(registry.get(NAME, null, null, Attributes.empty()));
   }
 
+  @Test
+  @SuppressWarnings("ReturnValueIgnored")
+  void getComponents_HighConcurrency() throws ExecutionException, InterruptedException {
+    List<Future<?>> futures = new ArrayList<>();
+    Random random = new Random();
+    int concurrency = 5;
+    ExecutorService executor = Executors.newFixedThreadPool(concurrency);
+
+    try {
+      for (int i = 0; i < 1000; i++) {
```

Comment:
with only 5 threads, do we really need 1000 executions to uncover the concurrency issue? Just wondering if 100 might be enough in this case.

Reply:
I looked closer and can repeat the error with close to 100% of the time with only 2 threads and 100 iterations, so I went ahead and reduced to those figures.